internal/util-collection/src/main/scala/sbt/internal/util/Settings.scala

```diff
@@ -8,14 +8,16 @@
 package sbt.internal.util
 
 import scala.language.existentials
-
 import Types._
 import sbt.util.Show
 
+import scala.annotation.tailrec
+
 sealed trait Settings[Scope] {
   def data: Map[Scope, AttributeMap]
   def keys(scope: Scope): Set[AttributeKey[_]]
   def scopes: Set[Scope]
+  def attributeKeys: Set[AttributeKey[_]]
```

Comment:
This is causing mima failure

```scala
[error]  * abstract method attributeKeys()scala.collection.immutable.Set in interface sbt.internal.util.Settings is present only in current version
[error]    filter with: ProblemFilters.exclude[ReversedMissingMethodProblem]("sbt.internal.util.Settings.attributeKeys")
```

Could you list this as an exception in build.sbt?